templates/docred/templates.yaml

```diff
@@ -0,0 +1,51 @@
+dataset: docred
+templates:
+  02af700c-e9e9-4a84-b75c-5fb29a5b7993: !Template
+    id: 02af700c-e9e9-4a84-b75c-5fb29a5b7993
+    jinja: "Read the following text and answer the questions.\n\nText:\n{% for sent\
+      \ in sents -%}\n{{ sent | join(\" \") }}\n{% endfor %} \n\nQuestion:\n\nFrom\
+      \ above text, find people, locations, organizations, time, numbers, and miscellaneous.\n\
+      |||\nPeople:\n{% for ners in vertexSet -%}\n{% for ner in ners if ner['type']\
+      \ == 'PER' -%}\n{{ner['name']}}{{\", \"}}\n{%- endfor %}\n{%- endfor %}\n\n\
+      Locations:\n{% for ners in vertexSet -%}\n{% for ner in ners if ner['type']\
+      \ == 'LOC' -%}\n{{ner['name']}}{{\", \"}}\n{%- endfor %}\n{%- endfor %}\n\n\
+      Organizations:\n{% for ners in vertexSet -%}\n{% for ner in ners if ner['type']\
+      \ == 'ORG' -%}\n{{ner['name']}}{{\", \"}}\n{%- endfor %}\n{%- endfor %}\n\n\
+      Time:\n{% for ners in vertexSet -%}\n{% for ner in ners if ner['type'] == 'TIME'\
+      \ -%}\n{{ner['name']}}{{\", \"}}\n{%- endfor %}\n{%- endfor %}\n\nNumbers:\n\
+      {% for ners in vertexSet -%}\n{% for ner in ners if ner['type'] == 'NUM' -%}\n\
+      {{ner['name']}}{{\", \"}}\n{%- endfor %}\n{%- endfor %}\n\nMiscellaneous:\n\
+      {% for ners in vertexSet -%}\n{% for ner in ners if ner['type'] == 'MISC' -%}\n\
+      {{ner['name']}}{{\", \"}}\n{%- endfor %}\n{%- endfor %}"
+    name: type-to-entity
```

Comment:
This template results in commas at the end of each type, e.g. `Locations: New Britain, New Ireland, Australia, Rabaul, Rabaul, Kavieng, Kavieng, Japan, Japanese, `. I think it's more natural to not have a comma at the end. You should be able to use the `join` jinja filter for this. Do you mind making this change?